simple-navmesh-constraint.js

```diff
@@ -21,7 +21,28 @@ AFRAME.registerComponent('simple-navmesh-constraint', {
       default: ''
     }
   },
-  
+
+  init: function () {
+    this.updateNavmeshEntities = this.updateNavmeshEntities.bind(this);
+  },
+
+  play: function () {
+    this.el.sceneEl.addEventListener('child-attached', this.updateNavmeshEntities);
+    this.el.sceneEl.addEventListener('child-detached', this.updateNavmeshEntities);
+  },
+
+  pause: function () {
+    this.el.sceneEl.removeEventListener('child-attached', this.updateNavmeshEntities);
+    this.el.sceneEl.removeEventListener('child-detached', this.updateNavmeshEntities);
+  },
```

Comment:
Instead of doing the register/unregister listener in play and pause, I would do it in init and remove, this way this would be properly updated if I had an entity while in editor mode. I have actually this use case.